### DISCUSSION TITLE
SVG file import reversed

### DISCUSSION THREAD

**Discussion by Ziggy2013** [arkypita/LaserGRBL#1374] [Q&A]
I just upgraded from v3.1.2 to 4.2.1.

In the latest version, when I import the same SVG file the x/y axis are both reversed. 

Also the SVG import is placed at least 200mm negative offset on the x axis.  This x axis negative offset is approx 3 times what the x axis positive offset was in version 3.1.2.  In fact, the larger negative offset in version 4.2.1 means that it is difficult to position the laser on the bed so that the SVG can be lasered.

Is this a new "feature" in later versions or a bug?

In the meantime I have no choice but to revert to 3.1.2.

EDIT : The new feature or bug first appeared in 3.3.1

Thanks

**Comment by Ziggy2013** [comment]
AFAICT the issue is with "viewbox" parameters in the svg file.

By grouping and saving in Inkscape the viewbox parameters can be eliminated from the xml. The svg file then imports ok.

**Reply by arkypita** [reply]
Sorry, I've written this several times but sadly the SVG format is a mess to manage. The support is incomplete and inconsistent, but it is what it is.

**Reply by Ziggy2013** [reply]
Thank you arkypita for making SVG support possible.

Even "as it is", the SVG support is very useful.
